semana15/sistema-bancario/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import express, { Request, Response } from 'express'
+import cors from 'cors'
+
+
+const app = express()
+app.use(express.json())
+app.use(cors())
+
+app.listen(3003, () => {
+    console.log('Servidor rodando na porta 3003!')
+})
+
+
+type conta = {
+    nome: string,
+    cpf: string,
+    dataNascimento: string,
+    saldo: number,
+    extratos: extrato[]
+}
+
+type extrato = {
+    valor: number,
+    data: string,
+    descricao: string
+}
+
+app.post("/contas", (req: Request, res: Response) => {
+    const conta = req.body as conta
```

Comment:
Aqui faltou apenas verificar a idade do usuário, dado que só devem ser criadas contas para maiores de 18 anos.
Para isso, crie uma variável que calcula a idade do usuário a partir da dataDeNascimento e a data atual (use o Date.now()) e em seguida faça uma condicional que verifica se esse valor é menor que 18. 
Se o valor for menor que 18, mande um `throw new Error()` para encerrar a requisição. Se for maior, o código apenas continua da forma que está escrito